src/pages/Admin/Admin.tsx

```diff
@@ -277,10 +277,16 @@ const EventsComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
     },
   ]);
 
-  const events = useSelector((state) => state.firestore.ordered.events);
+  let events = useSelector((state) => state.firestore.ordered.events);
   const [showCreateEventModal, setShowCreateEventModal] = useState(false);
   const [showDeleteEventModal, setShowDeleteEventModal] = useState(false);
   const [editedEvent, setEditedEvent] = useState<WithId<VenueEvent>>();
+  events?.sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
+  //filter out events that have already finished
+  events = events?.filter(
```

Comment:
Did Ed ask you to filter out the past ones? Feels a bit weird for the admin.

Reply:
No. I thought it would be good to avoid having a long list of past events that already happened at the top. I have commented it out for now and asked Ed.

Reply:
Yeah he said to just keep the upcoming ones